apps/sertra/pages/create.tsx

```diff
@@ -1,10 +1,10 @@
 import { GetStaticProps, NextPage } from "next";
-import { SWRConfig } from "swr";
-
+import { ApiURLs, TsApiURLs } from "../api/urls";
 import { getPackageList } from "../api/hooks";
-import { TsApiURLs } from "../api/urls";
+import { SWRConfig } from "swr";
```

Comment:
Please don't mess up [the import order](https://github.com/thunderstore-io/thunderstore-ui/pull/510/commits/618a1b07c383e9645e0652d83c0bbd28c89b6725).

Reply:
Oof looks like the merge switched it around, I'll fix.